tutorials/2_sprite_animations_gestures/code/lib/main.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flame/gestures.dart';
+import 'package:flame/sprite.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+
+void main() {
+  final myGame = MyGame();
+  runApp(
+    GameWidget(
+      game: myGame,
+    ),
+  );
```

Comment:
I think we can just inline all this, wdyt? save some lines of code.

Reply:
fixed